src/main/java/io/github/cottonmc/energy/api/DefaultEnergyTypes.java

```diff
@@ -34,12 +34,12 @@ public EmptyEnergyType() {
 
         }
         @Override
-        public int getMaximumPacketSize() {
+        public int getMaximumTransferSize() {
```

Comment:
Is there even a reason to keep EmptyEnergyType around if the EnergyPacket structure is gone?

Reply:
It's standard for all registries. Perhaps we should nix both the "formal" energy type registry and the empty energy type.

Reply:
Alternatively, we could consider the default energy type to be "LV"

Reply:
~~honestly I feel we should just keep EnergyPacket around since getting rid of it requires so much fundamental restructuring~~

Reply:
Be sure to take out the EmptyEnergyType class since it’s now unused.